forge/forge/tvm_to_python.py

```diff
@@ -810,6 +810,44 @@ def populate_vstack_args(graph, nid, compiler_cfg):
     args = [("slices", f"{slice_size}")]
     return args
 
+def populate_unsqueeze_args(graph, nid, compiler_cfg):
+    node = graph["nodes"][nid]
+    input_shape = graph["nodes"][node["inputs"][0][0]]["attrs"]["shape"][0][0]
+    output_shape = node["attrs"]["shape"][0][0]
+    
+    assert len(input_shape) + 1 == len(output_shape), "Unsqueeze only supports adding 1 dimension"
+
+    dim = -1
+    for i, (inp_dim, out_dim) in enumerate(zip(input_shape, output_shape)):
+        if inp_dim != out_dim:
+            dim = i
+            break
+
+    assert dim != -1
```

Comment:
Do we support negative (un)squeeze dims then? Also, let's add comments to all asserts just to give user a bit more details